context/app/static/js/components/Detail/Attribution/index.js

```diff
@@ -0,0 +1,3 @@
+import Attribution from './Attribution';
+
+export default Attribution;
```

Comment:
Thanks for doing this. It's most useful when there are multiple subsidiary components that can be all represented as separate files in a directory, with only one being exported. The benefit is hiding the details of the directory structure from the caller. Just want to be be sure the motivation makes sense, and it's not feeling arbitrary.